forms/OptionsetField.php

```diff
@@ -102,4 +102,19 @@ public function validate($validator) {
 	public function ExtraOptions() {
 		return new ArrayList();
 	}
+	
+	public function getAttributes() {
+		$attributes = parent::getAttributes();
+		unset($attributes['name']);
+		if(isset($attributes['required'])) {
+			unset($attributes['required']);
+		}
+		if(isset($attributes['role'])) {
+			unset($attributes['role']);
+		}
+		if(isset($attributes['aria-required'])) {
```

Comment:
I've done a bit of digging, and it seems (at first glance) that the correct thing to do is apply aria-required / required properties to the actual `<input />` elements.

http://jqueryvalidation.org/files/demo/radio-checkbox-select-demo.html

Maybe that's outside the scope of this fix though; Adding it to the fieldset would at least assist with screen-reader devices, which was addressed in my original link. :) It must be _somewhere_ on the formfield.